style.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap");
+
+:root {
+  --background-color: rgb(255, 255, 255);
+}
+
+html {
+  background-color: grey;
+  display: flex;
+  justify-content: center;
+}
```

Comment:
- [ ] The page has a margin at the top. It would be better to make our page fit the screen for a good user experience.
- suggestion: You can add the following lines before the `html` selector:
```css
* {
  margin: 0;
  padding: 0;
  box-sizing: content-box;
}
```

<img width="279" alt="image" src="https://user-images.githubusercontent.com/77190203/148279754-8783b9c3-05e2-42ac-ab1f-d0a548c81217.png">